src/BullOak.Repositories.EventStore.Test.Integration/StepDefinitions/StreamStepsDefinitions.cs

```diff
@@ -88,6 +88,12 @@ public async Task GivenISoft_DeleteTheStream()
         public async Task GivenIHard_DeleteTheStream()
             => await eventStoreContainer.HardDeleteStream(testDataContext.CurrentStreamId);
 
+        [Given(@"I soft-delete-by-event the stream")]
+        public async Task GivenI_Soft_Delete_by_EventTheStream()
```

Comment:
Could use 

```csharp
public Task GivenI_Soft_Delete_by_EventTheStream() => 
  eventStoreContainer.SoftDeleteByEvent(testDataContext.CurrentStreamId);
 ```

(no need for `async`/`await` here)

Reply:
Yes, it used to do more than that call (down to my lack of understanding how the test code captured session state) but now it doesn't. I'll change it.